src/domain/primitives/index.types.d.ts

```diff
@@ -13,6 +13,9 @@ type Pubkey = string & { [pubkeySymbol]: never }
 declare const walletIdSymbol: unique symbol
 type WalletId = string & { [walletIdSymbol]: never }
 
+declare const walletnameSymbol: unique symbol
+type Walletname = string & { [walletnameSymbol]: never }
```

Comment:
I'm wondering if we shouldn't use `WalletName` ie camel case

Reply:
It's an easy change if we'd like to do this now